src/openrct2/platform/Posix.cpp

```diff
@@ -488,6 +489,11 @@ bool platform_process_is_elevated()
 #    endif // __EMSCRIPTEN__
 }
 
+bool Platform::IsRunningInWine()
```

Comment:
This needs to be in Platform.Posix.cpp, in the same way it had to go into Platform.Win32.cpp.